### DISCUSSION TITLE
ACCOUNT SUSPENDED

### DISCUSSION THREAD

**Discussion by ghost** [community/community#142692] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Username: Aninooo
Ticket: Reinstatement Request #3067802

Dear GitHub Support Team,

I am writing to express my sincere apologies for any actions that may have violated your policies. I am uncertain about the specific violation, and I assure you that I will take the necessary steps to familiarize myself with all of GitHub’s policies moving forward.

My account is crucial for my work and personal projects, and I kindly request that you consider reinstating my account. My username is Aninooo, and my email associated with the account is bryanlomerioanino@gmail.com.

Thank you for your understanding, and I appreciate your attention to this matter.

Sincerely,
Aninooo

**Comment by ebndev** [comment]
Hi @AninoQt,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
